src/storage/src/read_object.rs

```diff
@@ -36,6 +37,31 @@ impl ReadObjectResponse {
         Self { inner }
     }
 
+    /// Create a ReadObjectResponse, given a data source.
+    ///
+    /// Use this API to mock the return type of
```

Comment:
s/Use this API/Use this method/

"API" is terribly easy to confuse with protocol level things.

Reply:
Fixed